main/main.cpp

```diff
@@ -353,7 +353,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	ClassDB::register_class<Performance>();
 	engine->add_singleton(Engine::Singleton("Performance", performance));
 
-	GLOBAL_DEF("debug/settings/crash_handler/message", String("Please include this when reporting the bug on https://github.com/godotengine/godot/issues"));
+	GLOBAL_DEF("debug/settings/crash_handler/message", "Godot version: " + get_full_version_string() + String("\nPlease include this when reporting the bug on https://github.com/godotengine/godot/issues"));
```

Comment:
I agree with adding the Godot version to the backtrace, but I'm not sure it's a good idea to make it part of the configurable crash handler message, as it means it will change at every new release, and users who customized this field will lose the ability to include the Godot version (as they can't call `get_full_version_string()` themselves in the configurable string).

So I'd simply hardcode this in the relevant crash handler code.

Reply:
I'll change that.

Reply:
Ping.